OneSchedule.Domain/StateMachine/GetState/GetEventMenuState.cs

```diff
@@ -0,0 +1,76 @@
+﻿using OneSchedule.Attributes;
+using OneSchedule.Data.Abstractions;
+using OneSchedule.Domain.Abstractions.StateMachine;
+using OneSchedule.Domain.Models;
+using OneSchedule.Entities;
+using System;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule.Domain.StateMachine.GetState
+{
+    [StateName("GetEventMenu")]
+    public class GetEventMenuState : IState
+    {
+        private readonly IRepository<EventEntity> _eventRepository;
+        private readonly ITelegramBotClient _bot;
+
+        public GetEventMenuState(IRepository<EventEntity> eventRepository, ITelegramBotClient bot)
+        {
+            _eventRepository = eventRepository;
+            _bot = bot;
+        }
+        public async Task HandleAsync(IStateContext stateContext, DtoDomain dtoDomain)
+        {
+            if (dtoDomain.MessageText.Contains("Save"))
```

Comment:
You have already implemented a strategy pattern. Why do you use these is else statements?